nautobot/extras/templates/extras/job_detail.html

```diff
@@ -44,6 +44,17 @@
                 <td>Job Class</td>
                 <td><code>{{ object.job_class_name }}</code></td>
             </tr>
+            <tr>
+                <td>Job Class Path</td>
+                <td>
+                    <span class="hover_copy">
+                        <span id="job_class_path_copy"><code>{{ object.source | split:"." | first }}/{{ object.module_name }}/{{ object.job_class_name }}</code></span>
```

Comment:
Can't we just use `{{ object.class_path }}` instead of manually constructing the class-path string here?

Reply:
Thanks, I updated. There was also this `<td>{{ object.source | split:"." | first | bettertitle }}</td>` that threw me off to begin with, I updated to `<td>{{ object.source | bettertitle }}</td>`.